packages/@remirror/react-social/src/components/social-editor-emoji.tsx

```diff
@@ -67,10 +68,6 @@ const EmojiDropdown = (props: Required<SocialEmojiState>) => {
 export const SocialEmojiComponent = () => {
   const { index, list, command } = useSocialEmoji();
 
-  if (!command) {
```

Comment:
This was the issue with the delayed positioner. For the `positioner` to calculate the position it needs to have an element to inspect. So the element needs to be persisted.